addons/l10n_es_edi_tbai/models/account_move.py

```diff
@@ -0,0 +1,237 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from datetime import datetime
+from re import sub as regex_sub
+
+from lxml import etree
+from odoo import api, fields, models
+from pytz import timezone
+from werkzeug.urls import url_quote
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    # Stored fields
+    l10n_es_tbai_is_required = fields.Boolean(
+        string="TicketBAI required",
+        help="Is the Bask EDI (TicketBAI) needed",
+        compute="_compute_l10n_es_tbai_is_required"
+    )
+    l10n_es_tbai_chain_index = fields.Integer(
+        string="Chain index (TicketBai)",
+        help="Invoice index in chain, set if and only if an in-chain XML was submitted and did not error",
+        store=True, copy=False, readonly=True,
+    )
+
+    # Relations
+    l10n_es_tbai_post_xml = fields.Many2one(
+        comodel_name="ir.attachment", readonly=True, copy=False,
+        string="Submission XML",
+        help="Submission XML sent to TicketBAI. Kept if accepted or no response (timeout), cleared otherwise."
+    )
+    l10n_es_tbai_cancel_xml = fields.Many2one(
+        comodel_name="ir.attachment", readonly=True, copy=False,
+        string="Cancellation XML",
+        help="Cancellation XML sent to TicketBAI. Kept if accepted or no response (timeout), cleared otherwise."
+    )
+
+    # Non-stored fields
+    l10n_es_tbai_sequence = fields.Char(string="TicketBAI sequence", compute="_compute_l10n_es_tbai_sequence")
```

Comment:
looks like the same as `sequence_number` on SequenceMixin

Reply:
It is a bit more complex, so it seems necessary to keep it.